### PR TITLE
Zaka

### PR DESCRIPTION
**Pull request by Codonisst**

### CODE REVIEW

File: layout/root/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Layout = ({ children }) => {
         // minHeight: "100vh",
         height: "100vh",
         maxHeight: "90vh",
-        backgroundImage: `url(${bg.src})`,
+        // backgroundImage: `url(${bg.src})`,
```
Comment (Mehmood11) [inline layout/root/index.js]: Root Layout k Changes remove ka

File: sections/dashboard/stepper/add-extra/index.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  // Add more options as needed
+];
+
+const AddExtra = () => {
+  const methods = useForm();
+
+  const { handleSubmit, watch } = methods;
+
+  const onSubmitHandler = (data) => {
+    const selectedValue = watch('services');
```
Comment (Mehmood11) [inline sections/dashboard/stepper/add-extra/index.js]: dalta de watch waly use kaly ? da remove ka

File: sections/dashboard/super-admin/super-admin-data.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    {
+      id: 1,
+      airLine: turkishAirline,
+      bookingId: "BK3663",
+      destination: <Box sx={{display:"flex", alignItems:"center"}}> BGY <EastIcon fontSize="12px" /> ATQ </Box>,
+      leadpax: "Baljeet Kaur",
+      departureDate: "12/12/23",
+      agentName: "Baljeet Kaur",
+      total: "76.98$",
+      status: <Button variant="contained" sx={{width:"95px", backgroundColor:"#629bf8", fontSize:"12px", fontWeight:"500", textTransform:"capitalize"}}>In Process</Button>,
```
Comment (Mehmood11) [inline sections/dashboard/super-admin/super-admin-data.js]: da ba dasi na e , simple ba yo string razi status k, 
status: 'pending' ya 'completed'.
Design hagha column wala array k jor ka.

File: sections/dashboard/stepper/add-extra/index.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    <Box>
+      <Typography
+        variant="h5"
+        component="h5"
+        sx={{ color: "#465365", fonSize: "12px", fontWeight: "600", my: 2 }}
+      >
+        Add Somthing{" "}
+        <Typography
+          variant="h5"
+          component="sapn"
```
Comment (Mehmood11) [inline sections/dashboard/stepper/add-extra/index.js]: speeling 'span'

**Review by Mehmood11 — COMMENTED** [review]
Check comments

File: sections/dashboard/stepper/add-extra/index.js (hunk 2 above)
Comment (Codonisst) [reply sections/dashboard/stepper/add-extra/index.js]: da washo

File: sections/dashboard/stepper/add-extra/index.js [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  { id: 1, label: "Option 1", value: "option1" },
+  { id: 2, label: "Option 2", value: "option2" },
+];
+
+const AddExtra = () => {
+  const methods = useForm();
+
+  const { handleSubmit, getValues  } = methods;
+
+  const onSubmitHandler = () => {
```
Comment (Mehmood11) [inline sections/dashboard/stepper/add-extra/index.js]: Submit k data parameter khapal razi hagha data console ka, getValues remove ka

File: sections/dashboard/stepper/add-extra/index.js [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+              />
+            </ListItem>
+            <ListItem>
+              <ListItemText primary="Each bag cannot weigh more than 23kgs" />
+            </ListItem>
+            <ListItem>
+              <ListItemText primary="Additional bags booked are non refundable" />
+            </ListItem>
+          </List>
+          {/* <ul
```
Comment (Mehmood11) [inline sections/dashboard/stepper/add-extra/index.js]: remove extra comments

File: sections/dashboard/super-admin/index.js [hunk 7]
```diff
@@ -1,123 +1,19 @@
...
-
 const SuperAdmin = () => {
+  const [searchParam, setSearchParam] = useState();
   return (
     <Box>
-      <Grid container spacing={2}>
+      <Grid container spacing={2} mb="40px">
         {cardData.map((item) => (
-          <Grid item xs={3}>
+          <Grid item xs={12} md={6} lg={3}>
```
Comment (Mehmood11) [inline sections/dashboard/super-admin/index.js]: key warning check ka console k

**Review by Mehmood11 — COMMENTED** [review]
Check again

**Review by Mehmood11 — APPROVED** [review]

**Pull request merged**